soh/soh/Enhancements/randomizer/randomizer.h

```diff
@@ -12,6 +12,136 @@
 #define NUM_NAVI_MESSAGES 19
 #define NUM_ICE_TRAP_MESSAGES 23
 
+typedef enum RandoOption {
```

Comment:
we put the other rando enums in `randomizerTypes.h` to avoid some issues we ran into with dependencies when including `randomizer.h`in some other files in the past, so it'd probably be a good idea to put this one in there too